pkg/controllers/management/auth/legacy_grb_cleaner.go

```diff
@@ -1,70 +0,0 @@
-package auth
-
-import (
-	"maps"
-	"slices"
-	"strings"
-
-	grbstore "github.com/rancher/rancher/pkg/api/norman/store/globalrolebindings"
-	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
-	"github.com/rancher/rancher/pkg/types/config"
-	"k8s.io/apimachinery/pkg/runtime"
-)
-
-type grbCleaner struct {
-	mgmt *config.ManagementContext
-}
-
-func newLegacyGRBCleaner(m *config.ManagementContext) *grbCleaner {
-	return &grbCleaner{
-		mgmt: m,
-	}
-}
-
-// sync cleans up all GRBs to drop cluster-scoped lifecycle handler finalizers
-func (p *grbCleaner) sync(key string, obj *v3.GlobalRoleBinding) (runtime.Object, error) {
-	if key == "" || obj == nil {
-		return nil, nil
-	}
-	if obj.Annotations[grbstore.GrbVersion] == "true" {
```

Comment:
Should you `grbstore.GrbVersion` and `roletemplate.RTVersion` (used as markers to avoid processing these objects again) be actively removed in the current grb and rt controllers?